requirements.txt

```diff
@@ -1,111 +1,80 @@
-# Wheel 0.25+ needed to install certain packages on CPython 3.5+
-# like Pillow and psycopg2
-# See http://bitly.com/wheel-building-fails-CPython-35
-# Verified bug on Python 3.5.1
-wheel==0.29.0
-
 # Django
-Django>=1.11.18,<2.0
+#Django>=1.11.18,<2.0
+Django==2.2.12
 
-django-filter==1.0.4
-django-rest-swagger==2.1.2
-djangorestframework==3.6.4
+django-filter==2.2.0
+drf-yasg==1.17.1
+djangorestframework==3.9.4
 djangorestframework-bulk==0.2.1
 
-# Simpl Modules
-simpl-users==0.7.7
+#simpl-users==0.7.6
+git+https://github.com/simplworld/simpl-users@6-django-2.0
 
```

Comment:
Needs to be updated to simpl-users v0.8.2, to support TokenAuthentication. Also, the 0.8 indicates the Simpl package supports Django 2.2.